parse-image-sha.py

```diff
@@ -0,0 +1,45 @@
+# This script pulls the docker images for dns containers and outputs the sha256
```

Comment:
Should we prepend the Apache license boilerplate here?

Reply:
Good idea, done.